front/src/applications/editor/components/EntitySumUp.tsx

```
@@ -61,6 +63,7 @@ async function getAdditionalEntities(
         try {
           return { [trackId]: await getEntity(infra, trackId, 'TrackSection', dispatch) };
         } catch (e) {
+          setFailure(castErrorToFailure(e));
```

is there a reason we were ignoring all these errors, and that what shouldn’t ignore them anymore ?

---

the new version requires now that we handle `e` in a catch. Instead of removing this rule, or just adding a console.error(e), @emersion suggested that it couldn't hurt to display the message to the user as well.

---

Right, but this code was maybe intentionally ignoring errors?

---

For some of them, there were  `TODO`s to remind if we wanted to display something or not. Not sure what we do then ?

---

Ok so after all, decided to keep the old behavior for the infra editor and not display anything to the user, just to devs to avoid the eslint warning